### PR TITLE
created essential minimum card component

### PR DESCRIPTION
**Pull request by noloman**

### CODE REVIEW

**Comment by noloman** [comment]
fixes #133

File: app/javascript/stories/ui/card.stories.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+// import { action } from '@storybook/addon-actions';
+// import { linkTo } from '@storybook/addon-links';
```
Comment (Diphendara) [inline app/javascript/stories/ui/card.stories.js]: May you need to delete the comment lines if you don't need it for work

File: app/javascript/bundles/UI/components/Card/Card.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
+.card {
+  border-radius: 10px;
+  border-width: 2px;
```
Comment (alvarocasadoc) [inline app/javascript/bundles/UI/components/Card/Card.scss]: I would change this to 3px

File: app/javascript/bundles/UI/components/Card/Card.scss [hunk 3]
```diff
@@ -0,0 +1,25 @@
+.card {
+  border-radius: 10px;
```
Comment (alvarocasadoc) [inline app/javascript/bundles/UI/components/Card/Card.scss]: Maybe 15px would look better

File: app/javascript/bundles/UI/components/Card/Card.scss [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  font-weight: bold;
+}
+
+.card-secondary {
+  color: #F2F2F2;
+  font-weight: bold;
+  margin-top: 5px;
+}
+
+.card-completed {
```
Comment (alvarocasadoc) [inline app/javascript/bundles/UI/components/Card/Card.scss]: I think we should rename this to card-footer to be more descripting

File: app/javascript/bundles/UI/components/Card/Card.scss [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  padding: 0.5em;
+  background-color: #FFFFFF;
+}
+
+.card-main {
+  color: #354093;
+  font-weight: bold;
+}
+
+.card-secondary {
```
Comment (alvarocasadoc) [inline app/javascript/bundles/UI/components/Card/Card.scss]: I think we should rename this to card-body to be more descripting

File: app/javascript/bundles/UI/components/Card/Card.scss [hunk 6]
```diff
@@ -0,0 +1,25 @@
+.card {
+  border-radius: 10px;
+  border-width: 2px;
+  border: 1px solid transparent;
+  border-color: #f2f2f2;
+  padding: 0.5em;
```
Comment (alvarocasadoc) [inline app/javascript/bundles/UI/components/Card/Card.scss]: I'd rise this padding to 1em

File: app/javascript/bundles/UI/components/Card/Card.jsx [hunk 7]
```diff
@@ -0,0 +1,17 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import css from './Card.scss';
+import classnames from 'classnames';
+
+const Card = ({ mainText, subText, completedText }) => {
+  return <div class='card card-main'>
```
Comment (alvarocasadoc) [inline app/javascript/bundles/UI/components/Card/Card.jsx]: I think we should rename card-secondary and card-completed to card-body and card-footer respectively. This way the HTML is more readable.

Also, I think we should do the same with the title. Maybe we could add: `<div class='card-header'>` to encapsulate the header.

Maybe `mainText`, `subText` and `completedText` should be renamed to follow the same naming.

**Comment by alvarocasadoc** [comment]
Looks great to me! If you don't have further questions, feel free to merge your PR :wink:

**Pull request merged**